docs/layouts/shortcodes/build_table.html

```diff
@@ -0,0 +1,42 @@
+<style type="text/css">
+  .tg  {border-collapse:collapse;border-spacing:0;}
+  .tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+    overflow:hidden;padding:10px 5px;word-break:normal;}
+  .tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+    font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+  .tg .tg-c3ow{border-color:black;text-align:center;vertical-align:top}
+  .tg .tg-0pky{border-color:black;text-align:left;vertical-align:top}
+  .tg .tg-dgl5{border-color:black;background-color:#FFF;text-align:left;vertical-align:top}
+  </style>
+  <table class="tg">
+  <thead>
+    <tr>
+      <th class="tg-c3ow" colspan="2"><span style="font-weight:bold">magic-modules - build commands</span></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td class="tg-0pky"><span style="font-weight:bold">make</span><br>runs make build</td>
+      <td class="tg-0pky"><span style="font-weight:bold">make build</span><br>  Builds the terraform provider with both mmv1 and tpgtools. Copies all generated files as a merge with the provider in OUTPUT_PATH.</td>
+    </tr>
+    <tr>
+      <td class="tg-0pky"><span style="font-weight:bold">make upgrade-dcl</span><br> A command to upgrade the version of the dcl. Used within the terraform provider and tpgtools.</td>
```

Comment:
upgrade-dcl should be documented in a new section about tpgtools when we add one rather than in the getting started guide. I'd prefer to keep that in a separate PR if that's all right.

Reply:
sounds good!